x-pack/plugins/dataset_quality/server/types/malformed_docs.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface MalformedDocs {
```

Comment:
**suggestion:** should this be derived from a runtime type using io-ts, that you can later use for encoding/decoding the server response. Also moving it to the common folder might suite better its purpose.

Reply:
Good one! Addressed in [f9f6fa3](https://github.com/elastic/kibana/pull/172462/commits/f9f6fa3a8704a2b4df301526b65098bafc65fcd6).